Promise_NonGeneric.cs

```diff
@@ -434,8 +434,8 @@ private void AddProgressHandler(Action<float> onProgress, IRejectable rejectable
         /// </summary>
         private void InvokeRejectHandler(Action<Exception> callback, IRejectable rejectable, Exception value)
         {
-//            Argument.NotNull(() => callback);
-//            Argument.NotNull(() => rejectable);
+            //            Argument.NotNull(() => callback);
+            //            Argument.NotNull(() => rejectable);
```

Comment:
I'm sorry CTRL+K,D messed up this comment. Though it was wrong in the first place.